packages/manager/apps/pci-private-network/src/pages/new/new.constants.ts

```diff
@@ -0,0 +1,48 @@
+import { z } from 'zod';
+
+export const VLAN_ID = {
+  default: 1,
+  min: 0,
+  max: 4000,
+};
+
+export const DEFAULT_CIDR = '10.{vlanId}.0.0/16';
+
+export const GATEWAY_HOURLY_PLAN_CODE = 'gateway.s.hour.consumption';
+
+const ipSchema = z.string().ip();
+const maskSchema = z
+  .number()
+  .min(9)
+  .max(29);
+
+export const NEW_PRIVATE_NETWORK_FORM_SCHEMA = z.object({
+  region: z.string().min(1),
+  isLocalZone: z.boolean(),
+  name: z.string().min(1),
+  defaultVlanId: z.number().optional(),
+  vlanId: z
+    .number()
+    .min(VLAN_ID.min)
+    .max(VLAN_ID.max)
+    .optional(),
+  subnet: z.object({
+    cidr: z.string().refine((value) => {
+      const [ip, mask] = value.split('/');
+      return (
+        ipSchema.safeParse(ip).success && maskSchema.safeParse(+mask).success
```

Comment:
why is there a "+" ?

Reply:
Here mask from the split is string. `zod maskSchema.safeParse` seems not convert it very well then I have to convert it manually to number with "+" operator. Otherwise it doesn't work